### PR TITLE
Update README.md in parallel

### PR DESCRIPTION
**Pull request by claudi**
parallel changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# TestRepo
+# Testing Repo
```
Comment (claudi-test) [inline README.md:1]: Please fill out this field

Comment (claudi-test) [reply README.md:1]: Oki

File: README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
-# TestRepo
```
Comment (claudi) [inline README.md:1]: Old text

File: README.md (hunk 1 above)
Comment (claudi) [inline README.md:1]: New and improved text

**Review by claudi — COMMENTED** [review]
Changes reviewed

**Pull request merged**